pkg/sql/sql_adapter.go

```diff
@@ -0,0 +1,41 @@
+package sql
+
+import (
+	"context"
+	"database/sql"
+)
+
+type StdSQLBeginner struct {
+	*sql.DB
+}
+
+type StdSQLTx struct {
+	*sql.Tx
+}
+
+// BeginTx converts the stdSQL.Tx struct to our Tx interface
+func (c *StdSQLBeginner) BeginTx(ctx context.Context, options *sql.TxOptions) (Tx, error) {
```

Comment:
How about making these value receivers instead of pointers? There should be no change performance-wise since the embedded values are pointers anyway, and it would make for a bit simpler construction of these. :)